packages/cli/test/dev/integration-2.test.ts

```diff
@@ -81,6 +81,13 @@ test(
   })
 );
 
+test(
+  '[vercel dev] test rewrites can change status code',
+  testFixtureStdio('test-rewrites', async (testPath: any) => {
+    await testPath(425, '/status-rewrite-425', 'Hello World');
```

Comment:
Lets move this into the previous test testFixtureStio on line 80 since this is going to create a deployment and both rewrites are in the same deployment.